sql/2021/http/http2_http3_usage_by_rank.sql

```diff
@@ -0,0 +1,53 @@
+# standardSQL
+# Percentiles of sites that main page over HTTP/2 or HTTP/3 by rank
+SELECT
+  client,
+  rank_grouping,
+  CASE
+    WHEN rank_grouping = 10000000 THEN 'all'
+    ELSE FORMAT("%'d", rank_grouping)
```

Comment:
Defer all formatting to the results sheet
```suggestion
    ELSE rank_grouping
```

Reply:
I'd agree with you, if I didn't already have a `rank_grouping` column (see line 5) so this is the text column of that (mainly so I can call the last bucket `all` so not trying to claim we have 10 millions websites, when we only have ~7 million) so see no harm in text formatting the other rank_groupings and makes it easier to see in BigQuery:

![image](https://user-images.githubusercontent.com/10931297/135934900-72b0198b-e284-4876-b053-b0ec6c06ffdd.png)